input/groups/armor.json

```diff
@@ -1826,32 +1776,87 @@
     },
     "Guild Wars 2: Secrets of the Obscure (SotO)": {
       "Astral Ward": [
-        11580, 11585, 11591, 11593, 11628, 11637, 11687, 11688, 11692, 11694, 11712, 11722, 11732, 11742, 11751, 11755, 11775, 11786
+        11580,
```

Comment:
If you have any preferred formating just say, I went with single lines because I had to pick one common format, this one happens to be the most comfortable one for manual editing; but then again, I'd say we're nearing a final version pretty soon, so the format won't matter that much.

Reply:
Mmm, there's actually a good argument for having an individual number per line, as it makes it easier to see changes. What I might do once we've finished this round of tweaks is run a script over the groups to make all the formatting consistent and sort the ids within each group. And then I can rerun the script after making changes to keep it that way.